### ISSUE TITLE
[Bug]: UI not working on jeedom homebridge

### ISSUE THREAD

**Issue by Arch30n** [homebridge-eufy-security/plugin#49]
### What happened?

The UI Wizard is not working on Homebridge plugin for Jeedom. I'm stuck on the wheel.
![image](https://user-images.githubusercontent.com/9718673/170858195-5fc06bbb-b65b-4cc0-bef8-fee864c4c0ff.png)

So I configured the plugin via JSON, but it seems that "enableCamera": true is not taken into account? so i can't use them.
My JSON: 
{
    "platform": "EufySecurity",
    "username": "********@*****.com",
    "password": "**********",
    "enableCamera": true,
    "country": "FR"
}

I also noticed the following error in the log attached.

### Device Type

other (UnSupported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

v1.4.x (Supported)

### NodeJS Version

v14 (Supported)

### Operating System

Debian 10+ (Supported)

### Relevant log output

```shell
[29/05/2022, 09:53:59] [Homebridge UI] [homebridge-eufy-security] /usr/lib/node_modules/homebridge-eufy-security/homebridge-ui/server.js:107
      this.config['country'] = body.country ??= 'US';
                                            ^^^

SyntaxError: Unexpected token '??='
    at wrapSafe (internal/modules/cjs/loader.js:1001:16)
    at Module._compile (internal/modules/cjs/loader.js:1049:27)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:1114:10)
    at Module.load (internal/modules/cjs/loader.js:950:32)
    at Function.Module._load (internal/modules/cjs/loader.js:790:12)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:75:12)
    at internal/main/run_main_module.js:17:47
```

**Comment by thieren** [comment]
HI @Arch30n 

It could be that your config wizard is not working because of cache issues with the browser. Try deleting the browser cache and/or another browser for testing. (see #1 for reference)
But most likely this is due to the error you encounter in your log. See below for the solution.

The `enableCamera` option is in the wrong place in your config. See my example:

   ```
{
   "platform": "EufySecurity",
   "username": "*****",
   "password": "*****",
   "country": "DE",
   "ignoreStations": [],
   "ignoreDevices": [],
   "cameras": [
      {
         "serialNumber": "*****",
         "enableCamera": false,
         "enableButton": true,
         "motionButton": true,
         "rtsp": false,
         "unbridge": true,
         "forcerefreshsnap": true,
          "useCachedLocalLivestream": true,
          "videoConfig": {
             "audio": true
          }
       }
   ],
   "pollingIntervalMinutes": 30,
   "hkHome": 1,
   "hkAway": 0,
   "hkNight": 3,
   "hkOff": 63,
   "enableDetailedLogging": true,
   "CameraMaxLivestreamDuration": 600,
   "cleanCache": true,
}
```

Also if your using a doorbell camera the setting is ignored I think.

Lastly the log error is fixed as mentioned in #44 as a workaround until this is pulled into the main version you can update your node version (everything above version 15 should work).

**Comment by Arch30n** [comment]
Thanks ! Installing node v16.15.0 lts do the job ;)
